pkg/kubectl/cmd/annotate.go

```diff
@@ -92,6 +93,7 @@ var (
 )
 
 func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
+	LoadTranslations()
```

Comment:
I'm sure at least one cobra maintainer would be interested to know what was possible...